yeat/workflow/__init__.py

```diff
@@ -51,8 +52,8 @@ def run_workflow(args):
 def resolve_paths(infile):
     data = json.load(open(infile))
     for sample in data["samples"].values():
-        for readtype, reads in sample.items():
-            sample[readtype] = get_resolved_paths(reads)
+        for key, value in sample.items():
+            sample[key] = value if key in DOWNSAMPLE_KEYS else get_resolved_paths(value)
```

Comment:
This code was updated, but it still needs some work. Changing `readtype, reads` to `key, value` reflects the wider scope of the function, but it also makes the code harder to understand. Also, the name of the function suggests that it's working only with paths, but that's no longer true.